pkg/cmd/api/api.go

```diff
@@ -85,6 +85,9 @@ func NewCmdApi(f *cmdutil.Factory, runF func(*ApiOptions) error) *cobra.Command
 			any value that contains %[1]s{...}%[1]s in quotes to prevent the shell from
 			applying special meaning to curly braces.
 
+			This command supports invoking preview-stage API endpoints/features, when
```

Comment:
```suggestion
			This command supports opting into previews, which are feature-flagged, experimental API endpoints or behaviors. The API expects opt-in via the %[1]sAccept%[1]s header with format %[1]sapplication/vnd.github.<preview-name>-preview+json%[1]s and this command facilitates that via %[1]s--preview <preview-name>%[1]s.
```

What do you think of something like this? Happy for you to adjust a little, just trying to provide a little more detail to the purpose and use of this thing, since the `api` command is such a thin wrapper over the real request/responses.

Reply:
Right. If I had a doc like that, I'd be half tempted to include it in the documentation...

Anyway, I'll fold in these suggestions and push. I'm quite happy with this as it aligns with my initial request.